### ISSUE TITLE
Other translations

### ISSUE THREAD

**Issue by dcxSt** [DSpeckhals/bible.rs#66]
Is it possible to have several other translations? For instance, the good news bible.

**Comment by DSpeckhals** [comment]
Yes it's possible, but it would take time to implement. I recently made changes that took out the hard-coding for Bible books based on the KJV, which should make it easier for others. The hardest part is getting a translation into the right format, populating the search index, and distributing it. Pull requests are welcome.

**Comment by dcxSt** [comment]
I'm taking a look at it, it might take a little while cause I'm busy and kindof a noob

**Comment by dcxSt** [comment]
I'm unable to run the Docker container for Bible.rs. Building with `docker build -t biblers .` works fine, but when I try to run it with `docker run -p 8080:8080 --rm -it biblers` I get this error. 

<img width="911" alt="Screen Shot 2023-01-27 at 10 12 05 PM" src="https://user-images.githubusercontent.com/21654151/215239175-8918bf92-51e3-48b5-bb54-ee8cc0e8fe73.png">

I'm using a mac. I've tried downloading sqlite and sqlite-utils but that didn't help. There's also a Stack Overflow post resembling this error, but I don't understand the solution... (I haven't used docker before)

**Comment by DSpeckhals** [comment]
Are you running an ARM based Mac (M1/M2)? If so, that could be an issue with the shared library. I don't have a Mac to test with.

**Comment by dcxSt** [comment]
Yes, I'm running on an M1 machine. I'm going to try troubleshooting, I'll let you know if I have any success

**Comment by passionate2023** [comment]
I'm facing issues with both windows and mac as well. Please help me with the steps. I have my own bible to feed the data. as well as own sermons to create another one to feed the data and build it

**Comment by passionate2023** [comment]
I recommend to carry out the testing via Virtual box and it easy to get the steps

1. https://www.youtube.com/watch?v=huIKDwdR5vw&ab_channel=Benjamin%27sTech
2. https://www.youtube.com/watch?v=1BOcED3RFGM&ab_channel=TopNotchProgrammer
3. https://www.youtube.com/watch?v=J-S_TvtIm5Y&ab_channel=ProgrammingKnowledge

**Comment by DSpeckhals** [comment]
Thanks, I'm aware of virtual machines. My development system is x86 Arch Linux, and I don't have the bandwidth to support Docker installations on non-Linux operating systems. If you give me the steps you took and the exact error message, though, I might be able to help.

**Comment by passionate2023** [comment]
Can you please add one more page for tamil bible search ?

**Comment by mangoose20026** [comment]
Wow...Tamil Bible search is available ? where i can find the tamil bible search link? Please share.

**Comment by tallmtt** [comment]
What would be needed for a different translation? 

- json or csv format?
- database data somehow?

**Comment by DSpeckhals** [comment]
The way I did it for the existing database were JSON files mentioned in the README, which I then converted to SQLite insert statements. Those are run as part of the application runtime setup.

If you are able to generate all the inserts similar to how I did the English KJV, you can create your own database file with whatever translation you desire.
